src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Net.Http.Json
+{
+    public static partial class HttpClientJsonExtensions
+    {
+        public static System.Threading.Tasks.Task<object?> GetFromJsonAsync(this System.Net.Http.HttpClient client, string requestUri, System.Type type, System.Text.Json.JsonSerializerOptions? options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Is there a reason this couldn't be `GetJsonAsync`. There's precedence to name these methods `Get*Async`:

* https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.getstreamasync?view=netcore-3.1
* https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.getstringasync?view=netcore-3.1

Reply:
`GetJsonAsync` sounds to me like I will get back the JSON payload, not the actual parsed value from it.

Reply:
Blazor's shipped an (experimental) `GetJsonAsync` API for close to 2 years now, and we haven't had users complain that the name was misleading or confusing. It's not a major hassle for users to migrate to this API, but it would be convenient to not have to.